packages/apex-node/src/tests/asyncTests.ts

```diff
@@ -295,23 +295,15 @@ export class AsyncTests {
     apexTestResultQuery += 'FROM ApexTestResult WHERE QueueItemId IN (%s)';
 
     const apexResultIds = testQueueResult.records.map(record => record.Id);
-    let formattedIds = '';
-    const queries = [];
 
     // iterate thru ids, create query with id, & compare query length to char limit
-    for (const id of apexResultIds) {
-      const newIds = addIdToQuery(formattedIds, id);
-      const query = util.format(apexTestResultQuery, `'${newIds}'`);
-
-      if (query.length > QUERY_CHAR_LIMIT) {
-        queries.push(util.format(apexTestResultQuery, `'${formattedIds}'`));
-        formattedIds = '';
-      }
-      formattedIds = addIdToQuery(formattedIds, id);
-    }
-
-    if (formattedIds.length > 0) {
-      queries.push(util.format(apexTestResultQuery, `'${formattedIds}'`));
+    const queries: string[] = [];
+    for (let i = 0; i < apexResultIds.length; i += QUERY_RECORD_LIMIT) {
+      const recordSet: string[] = apexResultIds
+        .slice(i, i + QUERY_RECORD_LIMIT)
+        .map(id => `'${id}'`);
+      const query: string = util.format(apexTestResultQuery, recordSet.join());
```

Comment:
```suggestion
      const query: string = util.format(apexTestResultQuery, recordSet.join(','));
```
nit, i know that the join param is omittable, but i like more explicitness

Reply:
`join()` defaults to spilling with a `,`, but I like being explicit too.  I'll add it in.